docker-compose.override.yml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+
+services:
+  frontend:
+    image: danlynn/ember-cli:latest
+    depends_on:
+      - backend
+    volumes:
+      - .:/myapp
+      - ./node_modules:/myapp/node_modules
+    tmpfs:
+      - /myapp/tmp
+    ports:
+      - 4200:4200
+      - 7020:7020
+      - 7357:7357
+    command: ember server --proxy=http://backend
```

Comment:
The frontend container should be the same as in production. No one is developing using a container ATM.

Reply:
Removed dev container